src/openstudio_lib/HotWaterEquipmentInspectorView.cpp

```diff
@@ -0,0 +1,220 @@
+  connect(this, &HotWaterEquipmentDefinitionInspectorView::toggleUnitsClicked, m_designLevelEdit, &OSQuantityEdit2::onUnitSystemChange);
+  mainGridLayout->addWidget(m_designLevelEdit, 3, 0);
+
+  // Watts Per Space Floor Area
+
+  label = new QLabel("Watts Per Space Floor Area: ");
+  label->setObjectName("H2");
+  mainGridLayout->addWidget(label, 2, 1);
+
+  m_wattsPerSpaceFloorAreaEdit = new OSQuantityEdit2("W/m^2", "W/m^2", "W/ft^2", m_isIP);
+  connect(this, &HotWaterEquipmentDefinitionInspectorView::toggleUnitsClicked, m_wattsPerSpaceFloorAreaEdit, &OSQuantityEdit2::onUnitSystemChange);
+  mainGridLayout->addWidget(m_wattsPerSpaceFloorAreaEdit, 3, 1);
+
+  // Watts Per Person
+
+  label = new QLabel("Watts Per Person: ");
+  label->setObjectName("H2");
+  mainGridLayout->addWidget(label, 2, 2);
+
+  m_wattsPerPersonEdit = new OSQuantityEdit2("W/person", "W/person", "W/person", m_isIP);
+  connect(this, &HotWaterEquipmentDefinitionInspectorView::toggleUnitsClicked, m_wattsPerPersonEdit, &OSQuantityEdit2::onUnitSystemChange);
+  mainGridLayout->addWidget(m_wattsPerPersonEdit, 3, 2);
+
+  // Fraction Latent
+
+  label = new QLabel("Fraction Latent: ");
+  label->setObjectName("H2");
+  mainGridLayout->addWidget(label, 4, 0);
+
+  m_fractionLatentEdit = new OSQuantityEdit2("", "", "", m_isIP);
+  connect(this, &HotWaterEquipmentDefinitionInspectorView::toggleUnitsClicked, m_fractionLatentEdit, &OSQuantityEdit2::onUnitSystemChange);
+  mainGridLayout->addWidget(m_fractionLatentEdit, 5, 0);
+
+  // Fraction Radiant
+
+  label = new QLabel("Fraction Radiant: ");
+  label->setObjectName("H2");
+  mainGridLayout->addWidget(label, 4, 1);
+
+  m_fractionRadiantEdit = new OSQuantityEdit2("", "", "", m_isIP);
+  connect(this, &HotWaterEquipmentDefinitionInspectorView::toggleUnitsClicked, m_fractionRadiantEdit, &OSQuantityEdit2::onUnitSystemChange);
+  mainGridLayout->addWidget(m_fractionRadiantEdit, 5, 1);
+
+  // Fraction Lost
+
+  label = new QLabel("Fraction Lost: ");
+  label->setObjectName("H2");
+  mainGridLayout->addWidget(label, 6, 0);
+
+  m_fractionLostEdit = new OSQuantityEdit2("", "", "", m_isIP);
+  connect(this, &HotWaterEquipmentDefinitionInspectorView::toggleUnitsClicked, m_fractionLostEdit, &OSQuantityEdit2::onUnitSystemChange);
+  mainGridLayout->addWidget(m_fractionLostEdit, 7, 0);
+
+  // Stretch
+
+  mainGridLayout->setRowStretch(8, 100);
+
+  label =
+    new QLabel("<i>The object models hot water equipment in the zone which consumes district heating, such as cooking equipment or process loads. "
+               "All of the energy consumed by the equipment becomes a heat gain in the zone or is lost (exhausted). "
+               "This object consumes district heating energy directly and does not cause a load on a hot water plant loop or water heater. "
+               "For domestic hot water uses, such as sinks and showers, see WaterUse:Equipment.</i>");
+  label->setWordWrap(true);
+  mainGridLayout->addWidget(label, 9, 0, 1, 4);
```

Comment:
Add a footnote clarifying what the object is. There is no precedent, so I'd like people to weight in on whether this is helpful or not, anbd whether it should be displayed differently in the UI.

Made it look like this

![image](https://github.com/user-attachments/assets/76ab6450-2754-48c5-b180-e14a3611e535)


@macumber @marccosta @chrisbalbach @craigers290  @antonszilasi

Reply:
Yeah, I was looking at the IDD and I/O Reference trying to figure out why this wasn't added in the first place.  It is a bit confusing that it doesn't hook up to a water loop.  I think the text is helpful and looks pretty good.  If we wanted to be super fancy, we could add a "for more information" link to the I/O Reference.  That could be a general pattern but it might be too hard to maintain though.

Reply:
It's really the dhw equivalent of the ZoneHVACIdealLoadsAirSystem.

I agree that I don't want to keep external links up to date.